src/schemas/json/license-report-config.json

```diff
@@ -28,18 +29,18 @@
       }
     },
     "delimiter": {
-      "delimiter": ",",
+      "default": ",",
       "description": "CSV output format",
       "type": "string"
     },
-    "escapeCsvFilelds": {
+    "escapeCsvFields": {
       "default": "false",
       "description": "CSV output format: escape fields containing delimiter character",
       "type": "boolean"
     },
     "only": {
       "default": "null",
-      "description": "export deps. or deps/dev-/opt-/peer- deps. falsey -> output everything",
+      "description": "export deps. or deps/dev-/opt-/peer- deps. falsy -> output everything",
```

Comment:
I would just keep the original spelling as it seems to be acceptable too (I am no native speaker so my "wisdom" comes from [mdn](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)